codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/ServerCargoDependency.kt

```diff
@@ -17,9 +17,9 @@ import software.amazon.smithy.rust.codegen.smithy.RuntimeConfig
 object ServerCargoDependency {
     val AsyncTrait: CargoDependency = CargoDependency("async-trait", CratesIo("0.1"))
     val FuturesUtil: CargoDependency = CargoDependency("futures-util", CratesIo("0.3"))
+    val FormUrlEncoded: CargoDependency = CargoDependency("form_urlencoded", CratesIo("1"))
```

Comment:
This matches the version picked by `serde_urlencoded = "0.7"`.

https://github.com/nox/serde_urlencoded/blob/master/Cargo.toml#L21